### PR TITLE
notifi-react: milestone#1: context ready

### PR DESCRIPTION
**Pull request by eric-notifi**
- [x] Describe the purpose of the change
The notifi-react package's new context structure ready  
- [ ] Create test cases for your changes if needed
No need
- [x] Include the ticket id if possible (Collaborator only)
MVP-4297 milestone#1

### CODE REVIEW

File: packages/notifi-card-modal-example/cypress/component/NotifiSubscriptionCard.cy.tsx
Comment (eric-notifi) [inline packages/notifi-card-modal-example/cypress/component/NotifiSubscriptionCard.cy.tsx:1]: Cypress section is still just dummy scaffolding structure. Plan on implement it in the last milestone (ending section)

File: packages/notifi-card-modal-example/cypress/support/commands.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  //       items: Record<string, any>,
+  //       fixtureName?: string,
+  //     ): void;
+  //     mountNotifiSubscriptionCard(): void;
+  //     overrideFetchDataTargetGroupWithFixture(fixtureName: string): void;
+  //     overrideUserSettingsWithFixture(fixtureName: string): void;
+  //   }
+  // }
+}
+
```
Comment (eric-notifi) [inline packages/notifi-card-modal-example/cypress/support/commands.tsx:31]: Same comment as above

File: packages/notifi-card-modal-example/cypress/support/commands.tsx
Comment (eric-notifi) [inline packages/notifi-card-modal-example/cypress/support/commands.tsx:1]: Same comment as above

File: packages/notifi-card-modal-example/cypress/support/component.ts
Comment (eric-notifi) [inline packages/notifi-card-modal-example/cypress/support/component.ts:1]: Same comment as above

**Review by hans-notifi — APPROVED** [review]

**Pull request merged**